relation_engine/database/test/batch_time_travel_test.py

```diff
@@ -0,0 +1,56 @@
+# TODO start a new arango instance as part of the tests so:
+# a) we remove chance of data corruption and 
+# b) we don't leave test data around
+
+# TODO more tests
+
+from relation_engine.database.batch_time_travelling_db import ArangoBatchTimeTravellingDB
+from arango import ArangoClient
+from pytest import fixture
+
+HOST = 'localhost'
+PORT = 8529
+DB_NAME = 'test_timetravel_delta_batch_load_db'
+
+@fixture
+def arango_db():
+    client = ArangoClient(protocol='http', host=HOST, port=PORT)
+    sys = client.db('_system', 'root', '', verify=True)
+    sys.delete_database(DB_NAME, ignore_missing=True)
+    sys.create_database(DB_NAME)
+    db = client.db(DB_NAME)
+
+    yield db
+
+    sys.delete_database(DB_NAME)
+
+def test_get_vertex(arango_db):
+    col_name = 'verts'
+    arango_db.create_collection(col_name)
+    col = arango_db.collection(col_name)
+
+    # it's assumed that given an id and a timestamp there's <= 1 match in the collection
+    col.import_bulk([{'_key': '1', 'id': 'foo', 'created': 100, 'expires': 600},
```

Comment:
Test for get_vertex on 'bar' at 100? Throws an error right?

Reply:
That'll get the node, the range is inclusive. 99 would return `None`.

Reply:
Sorry, me type bad. I meant to say to add another 'bar' active at 100 to test that the error is thrown

Reply:
done, although used 200